bridge/opencensus/metric_test.go

```diff
@@ -32,6 +33,136 @@ import (
 	"go.opentelemetry.io/otel/sdk/resource"
 )
 
+func TestMetricProducer(t *testing.T) {
+	now := time.Now()
+	for _, tc := range []struct {
+		desc      string
+		input     []*ocmetricdata.Metric
+		expected  metricdata.ScopeMetrics
+		expectErr bool
+	}{
+		{
+			desc: "empty",
```

Comment:
I don't think this would be fixed in this PR given the producer comes from `"go.opencensus.io/metric/metricproducer"`, but just a heads up that returning empty data sets without metrics caused crashes on the collector: https://github.com/open-telemetry/opentelemetry-go/issues/3394

Reply:
This now returns nil for an empty batch.